txpipe/twopoint_fourier.py

```diff
@@ -221,11 +223,85 @@ def load_maps(self):
         syst_nc = None
         syst_wl = None
 
+        
+        # Load HEALPix systematics maps
+        syst_nc = self.config['syst_nc']
+        syst_wl = self.config['syst_wl']
+        if syst_nc:  
+            print('Deprojecting systematics maps for number counts')
+            n_systmaps = 0
+            s_maps = []
+            systmaps_dir = self.config['systmaps_nc_dir']
+            systmaps_path = pathlib.Path(systmaps_dir)
+            try:
+                for systmap in systmaps_path.iterdir():
+                    if systmap.is_file():
+                        systmap_file = str(systmap)
+                        print('Reading nc systematics map file: ', systmap_file)
+                        syst_map = healpy.read_map(systmap_file,verbose=False)
+                        
+                        # Find value at given ra,dec
+                        ra = 55.
+                        dec = -30.
+                        theta = 0.5 * np.pi - np.deg2rad(dec)
+                        phi = np.deg2rad(ra)
+                        nside = healpy.pixelfunc.get_nside(syst_map)
+                        ipix = healpy.ang2pix(nside, theta, phi)
+                        print('Syst map: value at ra,dec = 55,-30: ', syst_map[ipix])
+
+                        # normalize map for Namaster
+                        # set pixel values to value/mean - 1
+                        syst_map_mask = syst_map != healpy.UNSEEN
+                        mean = np.mean(syst_map[syst_map_mask]) # gives mean of all pixels with mask applied
+                        if mean != 0:
+                            print('Syst map: mean value = ', mean)
+                            syst_map[~syst_map_mask] = 0 # sets unmasked pixels to zero 
+                            syst_map = syst_map / mean - 1
+                            print('Syst map', systmap_file, 'normalized value at ra,dec = 55,-30: ', syst_map[ipix])
+                        
+                        s_maps.append(syst_map)
+                        n_systmaps += 1
+            except FileNotFoundError:
+                print('Systematics map file not found in path', systmap_path)
+                raise
+                
+            print('Number of systematics maps read: ', n_systmaps)
+            # We assume all systematics maps have the same nside
+            nside = healpy.pixelfunc.get_nside(syst_map)
+            npix = healpy.nside2npix(nside)
+            # needed for NaMaster:
+            s_maps_nc = np.array(s_maps).reshape([n_systmaps, 1, npix])
+
+# wl not implemented yet
```

Comment:
A good question!  Maybe look for pairs of maps?
We could also read a text file of paths to deproject, and look for pairs for shear.  That can wait, though.

Reply:
Yes, we could use ellipticity e1, e2. But will leave lensing systematics out for now.